package.json

```diff
@@ -1,33 +1,27 @@
 {
+  "name": "sdk-boilerplate",
   "author": "metaversecloud.com | topia.io",
   "description": "",
   "dependencies": {
     "@rtsdk/topia": "^0.8.5",
-    "axios": "^1.2.1",
-    "body-parser": "^1.20.1",
-    "concurrently": "^7.6.0",
+    "concurrently": "^8.2.2",
     "cors": "^2.8.5",
-    "dotenv": "^16.0.3",
+    "dotenv": "^16.4.5",
     "express": "^4.18.2",
     "yalc": "^1.0.0-pre.53"
   },
-  "engines": {
-    "node": "18.14"
-  },
   "keywords": [],
   "license": "ISC",
-  "main": "index.js",
-  "name": "sdk-boilerplate",
+  "main": "./server/index.js",
+  "devDependencies": {
+    "prettier": "3.2.5"
+  },
   "private": true,
   "scripts": {
-    "build": "npm run build -w client",
-    "build-dev": "npm run build",
-    "link-sdk": "yalc link @rtsdk/topia",
-    "unlink-sdk": "yalc remove @rtsdk/topia",
-    "server": "node server/index.js",
-    "client": "npm start --prefix client",
-    "start": "npm start -ws",
```

Comment:
We are missing the start script. Make sure `npm start` is working, and starting the server.

`http://localhost:3000` should work

Reply:
@fabioalxk I added the scripts but `npm start` is not working because of a `npm ERR! No workspaces found!`.